airbyte-integrations/connectors/source-gcs/source_gcs/stream_reader.py

```diff
@@ -50,7 +53,8 @@ def _initialize_gcs_client(self):
             raise ValueError("Source config is missing; cannot create the GCS client.")
         if self._gcs_client is None:
             credentials = self._get_credentials()
-            self._gcs_client = storage.Client(credentials=credentials)
+            # using default project to avoid getting project from env
```

Comment:
please specify that we need this for Oauth credentials only.

btw, I think we can use project from service account if its specified

Reply:
updated